.gitignore

```diff
@@ -11,3 +11,7 @@ Release
 Debug
 Benchmark
 vs2010.*
+autom4te.cache/
+config.log
+config.status
+!configure
```

Comment:
Agree that a generated script like configure shouldn't be in the repo. A typo, I think.